jump/src/installer.rs

```diff
@@ -245,42 +246,19 @@ impl<'a> Installer<'a> {
                     0
                 }
                 FileEntry::ScieTote((tote_file, entries)) => {
-                    let mut scie_tote: Option<TempDir> = None;
-                    let mut scie_tote_src = || {
-                        if let Some(tempdir) = scie_tote.as_ref() {
-                            return Ok::<_, String>(tempdir.path().join(&tote_file.name));
-                        }
-                        let scie_tote_tmpdir = TempDir::new().map_err(|e| {
+                    let mut scie_tote = ZipArchive::new(Cursor::new(
+                        &self.payload[location..(location + tote_file.size)],
+                    ))
+                    .map_err(|e| format!("{e}"))?;
+
+                    for (file, dst) in entries {
+                        let entry = scie_tote.by_name_seek(&file.name).map_err(|e| {
```

Comment:
Per zip upgrade note above, the `by_name_seek` method here is new. Used to just offer `by_name` which was `Read` but not `Seek`.